packages/js/src/shared-admin/components/ai-generate-titles-and-descriptions-upsell.js

```diff
@@ -70,41 +92,42 @@ export const AiGenerateTitlesAndDescriptionsUpsell = ( { learnMoreLink, thumbnai
 			</div>
 			<div className="yst-mt-6 yst-text-xs yst-font-medium">
 				<span className="yst-introduction-modal-uppercase">
-					{ sprintf(
-						/* translators: %1$s expands to Yoast SEO Premium. */
-						__( "New to %1$s", "wordpress-seo" ),
-						"Yoast SEO Premium"
-					) }
+					{ newToText }
 				</span>
 				&nbsp;
 				<span className="yst-uppercase yst-text-slate-700">21.0</span>
```

Comment:
This needs to be there only when displaying "New to Yoast SEO Premium"

Reply:
Right!